draft-ietf-rtcweb-jsep.xml

```diff
@@ -1778,19 +1778,20 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           "a=rtcp-fb" attributes, indicating its willingness to support
           "(UDP,TCP)/TLS/RTP/SAVPF". In order to simplify compatibility
           with such endpoints, JSEP implementations MUST follow the
-          following rules when processing the media m= sections in an
-          offer:</t>
+          following rules when processing the media m= sections in a
+          received offer:</t>
 
           <t>
             <list style="symbols">
-
-              <t>The profile in any "m=" line in any answer MUST
-              exactly match the profile provided in the offer.</t>
-
-              <t>Any profile matching the following patterns MUST be
-              accepted: "RTP/[S]AVP[F]" and
-              "(UDP/TCP)/TLS/RTP/SAVP[F]"</t>
-
+              <t>Any profile in the offer matching the following patterns
+              MUST be accepted: "RTP/[S]AVP[F]" and 
+              "(UDP/TCP)/TLS/RTP/SAVP[F]".
```

Comment:
this pattern could be read as matching "UDP/TCP/TLS/RTP/SAVPF"  I think that you want this "RTP/[S]AVP[F]" and "{UDP|TCP}/TLS/RTP/SAVP[F]" (that is, use braces for the first grouping and a vertical bar instead of a slash).